lib/onetime/app/api/base.rb

```diff
@@ -63,7 +63,6 @@ def check_locale! locale=nil
         locales << OT.conf[:locales].first                              # Ensure at least one configured locale is available
         locales = locales.uniq.reject { |l| !OT.locales.has_key?(l) }.compact
         locale = locales.first if !OT.locales.has_key?(locale)           # Default to the first available
-        OT.ld [:locale, locale, locales, req.env['rack.locale'], OT.locales.keys].inspect
         req.env['ots.locale'], req.env['ots.locales'] = (@locale = locale), locales
```

Comment:
Consider adding a comment or docstring to explain the purpose of the `check_locale!` method and how it works. This will make the code more self-documenting and easier for other developers to understand. [medium]

Reply:
There already were docs for it so I pushed an update anyway for clarity. 🙏